app/controllers/response_templates_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ResponseTemplatesController < ApplicationController
+  after_action :verify_authorized, except: %i[index]
+
+  def index
```

Comment:
I'm not that big of a fan of this logic, but since it worked previously I decided to keep it for now. Will spend some time to refactor this in the cycle.